tests/test_torch.py

```diff
@@ -489,8 +489,9 @@ def test_sequence_net():
 
 
 def test_multi_net(wandb_init_run):
-    net = ConvNet()
-    graphs = wandb.watch((net, net))
+    net1 = ConvNet()
+    net2 = ConvNet()
+    graphs = wandb.watch((net1, net2))
     output = net.forward(dummy_torch_tensor((64, 1, 28, 28)))
```

Comment:
net1?

Reply:
We would blow up with these changes because it would try to hook the torch model twice so I needed to create two instances...

Reply:
right, my comment was that net doesnt exist so you cant do `net.forward()`

Reply:
Yeah, was just able to confirm that by my failing test 🤣